core/objects.go

```diff
@@ -741,6 +741,13 @@ type SignedCertificateTimestamp struct {
 	LockCol int64
 }
 
+// RPCSignedCertificateTimestamp is a wrapper around SignedCertificateTimestamp
+// so that it can be passed through the RPC layer properly. Without this wrapper
+// the UnmarshalJSON method below will be used when marshaling/unmarshaling the
+// object, which is not what we want as it is not symmetrical (as it is intended
+// to unmarshal a rawSignedCertificateTimestamp into a SignedCertificateTimestamp)
+type RPCSignedCertificateTimestamp SignedCertificateTimestamp
```

Comment:
I know this takes us back to what we had before, but perhaps we shouldn't be using SignedCertificateTimestamp.UnmarshalJSON for this purpose? It seems like we ought to write a conversion function similar to the db-object-to-rpc-object ones.

It'd be nice to have a bug for that.